src/envoy/mixer/integration_test/stress_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"sync/atomic"
+	"testing"
+	"time"
+)
+
+const (
+	concurrent         = 10
+	duration_in_second = 30
+)
+
+func TestStressEnvoy(t *testing.T) {
+	// Not cache cache, enable quota
+	s, err := SetUp(t, basicConfig+","+quotaConfig, true)
+	if err != nil {
+		t.Fatalf("Failed to setup test: %v", err)
+	}
+	defer s.TearDown()
+
+	url := fmt.Sprintf("http://localhost:%d/echo", ClientProxyPort)
+
+	var count uint64 = 0
+	for k := 0; k < concurrent; k++ {
+		go func() {
+			for true {
+				if _, err := http.Get(url); err != nil {
```

Comment:
is this creating 1 connection per thread?

Can you use `wrk` to perform the actual stress test? 
"https://storage.googleapis.com/istio-build-deps/wrk-linux"